### ISSUE TITLE
Linker SuperCard DSTWO Plus

### ISSUE THREAD

**Issue by borrougagnou** [Plailect/Guide#1463]
Bonjour,

Je vois sur la page de https://3ds.guide/fr_FR/ntrboot
qu'il n'y a pas la SuperCard DSTWO Plus qui gère le mode 3DS maintenant

a vrai dire ils sont discret sur le sujet aussi, voici le lien de leurs site:
http://eng.supercard.sc/

**Comment by Ev1lbl0w** [comment]
Hey, firstly if you can, please translate your issue to english, it will be easier for everyone to understand.

From what i could understand (i'm not very good at french, correct me if i'm wrong), you're saying the guide should include the SuperCard DSTWO Plus, since it also has a 3DS mode, although the team is kinda "discreet".

Well, there is a good reason for that: there were reports of people using the 3DS mode having their consoles bricked because it actually emulates [Gateway](http://www.gateway-3ds.com/), and they were very unhappy with it. So, starting from version 3.5.1, the Gateway team decide to fully brick (and it was not an easy brick) 3DS's who were using their software from another flashcards. This, of course, sparked controversy and hate towards Gateway (which I totally agree, they're willing to risk truthfully customers and are downright "destroying" consoles for people who are mostly inoccent).

Bottom line, anyone who is not aware of this can very easily get a 3DS bricked. That's probably the reason this flashcard was not included in the list.

**Comment by borrougagnou** [comment]
Ho ! sorry for this inconveignant

and Gateway brick console? strange, but that okay.
After I don't follow the news about Linker :/
